src/modules/refreshScores.js

```diff
@@ -0,0 +1,13 @@
+// /* eslint-disable no-restricted-globals */
+import displayScores from './displayScores.js';
+import getScores from './getScores.js';
+// acquire scores after refresh
+const refresh = () => {
```

Comment:
- [ ] Well, when it comes to your `refresh` button it looks like it's not working. I have tested with console.log still no action. Please make it work based on project requirement.